### PR TITLE
Update CONTRIBUTING.md and how to work with current and next releases

### PR DESCRIPTION
**Pull request by johannes-b**
Signed-off-by: Johannes <johannes.braeuer@dynatrace.com>

Update of contributing guide to explain how to work on new or released documentation

### CODE REVIEW

**Review by StackScribe — APPROVED** [review]
lgtm although I wonder if the new section should be after the Fork/Clone section but before the running Hugo locally section.

Also, how does one build the docs locally to view the documentation in the development directory (e.g. 0.14.x in this case)?  The best I can figure out is to save a copy of config.toml, then remove the 0.14.x from the file while I am writing.  Then restore the original file before pushing the PR.  Is there a more elegant solution?

**Comment by afzal442** [comment]
I wondered how one could sync feature branch with the base repo in case of resolving conflict in files? I think that we can add sth like, `git rebase master` or `git merge master` in that case.
For ref: https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md#make-a-pull-request

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,13 +36,33 @@ If you see a problem and are unable to fix it yourself or have an idea for an en
...
+
+If you want to work on an issue or enhancement, two questions arise: 
+1. Does it relate to a *new feature* that is not yet released? 
+1. Does it relate to an *already released* Keptn version?
+
+Currently, the Keptn project follows the approach outlined below. For clarification, an example is used in which the Keptn project recently released `0.13.x` and is currently working on `0.14.x`. These release versions will of course increase when the release train moves on. 
+
+**Documentation for new features**
+
+For the new `0.14.x` release, a folder in [./content/docs](https://github.com/keptn/keptn.github.io/tree/master/content/docs) is available, which receives documentation for new features or breaking changes. This folder is *hidden*  and not publically rendered. This is controlled with the `ignoreFiles` flag in [config.toml](https://github.com/keptn/keptn.github.io/blob/master/config.toml).
```
Comment (oleg-nenashev) [inline CONTRIBUTING.md:49]:
```suggestion
For each recent release (e.g. `0.14.x`), a folder in [./content/docs](https://github.com/keptn/keptn.github.io/tree/master/content/docs) is available, which receives documentation for new features or breaking changes. This folder is *hidden*  and not publically rendered. This is controlled with the `ignoreFiles` flag in [config.toml](https://github.com/keptn/keptn.github.io/blob/master/config.toml). 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -35,13 +36,33 @@ If you see a problem and are unable to fix it yourself or have an idea for an en
...
+> Update the content in the corresponding folder of the Keptn release **and** in the current folder that is in development. Consequently, your PR will target at least two folders if the change is relevant for the upcoming releases too.
+
 ## Fork and clone the repository
 
 Perform the following steps to create a copy of this repository on your local machine:
 
 1. Fork the Keptn repository:
 
-     - Log into github (or create a github account and then log into it).
+     - Log into Github (or create a GitHub account and then log into it).
```
Comment (oleg-nenashev) [inline CONTRIBUTING.md:65]:
```suggestion
     - Log into GitHub (or create a GitHub account and then log into it).
```

**Review by oleg-nenashev — APPROVED** [review]
Just a few suggestions. LGTM overall

File: CONTRIBUTING.md (hunk 1 above)
Comment (johannes-b) [reply CONTRIBUTING.md:49]: Addressed it. Thx!

**Review by StackScribe — APPROVED** [review]
Looks great!  It still doesn't tell me how to build 0.14.x locally but I guess people who work in an IDE may not care and I have figured out a little kludge that works.  Let's merge this and make the info available to everyone!

**Pull request merged**